assets/shaders/all.wgsl

```diff
@@ -82,13 +82,12 @@ fn object_distance(p: vec3<f32>, sdf_index: i32) -> f32 {
     }
 
     // Correct the returned distance to account for the scale
-    // Note that this is not perfect yet and only seems to work when scaling
-    // uniformly (e.g. not scaling on axis).
-    // TODO find scale from transform matrix and correct for it again.
-    // previous factor was:
-    // length(sdf_scale) / length(vec3(1.0))
-    // Or find jacobian?
-    return d_current_object;
+    // Note that this is not perfect yet.
+    let scale = vec3<f32>(length(inverse_transform[0].xyz),
```

Comment:
If that part is wrong, I blame chatgpt. At least it make the eyes look much better on default duck.